tests/display.py

```diff
@@ -159,6 +159,13 @@ def testImageTypes(self):
 
             im = afwImage.MaskU(self.fileName, 3)
             dummy.mtv(im)
+    
+    def testInteract(self, timeout=1):
```

Comment:
This was corrected in a later commit. I rebased before pushing my commits but it didn't seem to fix this (but the current file [https://github.com/lsst/afw/blob/master/python/lsst/afw/display/interface.py](url) shows timeout removed).